portals/publisher/src/main/webapp/site/public/locales/en.json

```
@@ -17,6 +17,30 @@
       "value": "Cancel"
     }
   ],
+  "Actions": [
+    {
+      "type": 0,
+      "value": "Actions"
+    }
+  ],
+  "Active.Development.Available": [
```

Would be helpful to change the key to match with the value. 

```suggestion
  "Active.Deployment.Available": [
```

---

Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)

---

Addressed requested changes by [b341c7d](https://github.com/wso2/apim-apps/pull/515/commits/b341c7d820fee359e17f7a668acbf4ec92d7fcd7)